Libraries/Components/ScrollResponder.js

```diff
@@ -471,6 +472,15 @@ var ScrollResponderMixin = {
    * @platform ios
    */
   scrollResponderFlashScrollIndicators: function() {
+    if (Platform.OS === 'android') {
```

Comment:
We could just get rid of the iOS code path and always use `UIManager.dispatchViewManagerCommand`, afaik it also works on iOS.